codebuild/spec/buildspec_fuzz_scheduled.yml

```diff
@@ -77,12 +33,20 @@ phases:
         cmake . -Bbuild \
         -DCMAKE_PREFIX_PATH=/usr/local/$S2N_LIBCRYPTO \
         -DS2N_FUZZ_TEST=on \
-        -DFUZZ_TIMEOUT_SEC=27000
+        -DFUZZ_TIMEOUT_SEC=27000 \
+        -DCOVERAGE=on \
+        -DBUILD_SHARED_LIBS=ON
       - cmake --build ./build -- -j $(nproc)
   post_build:
     on-failure: ABORT
     commands:
       # -L: Restrict tests to labels matching the pattern 'fuzz'
-      # -R: Run the single fuzz test defined in ${FUZZ_TESTS}
       # --timeout: override ctest's default timeout of 1500
-      - cmake --build build/ --target test -- ARGS="-L fuzz -R ${FUZZ_TESTS} --output-on-failure --timeout 28800"
+      - cmake --build build/ --target test -- ARGS="-L fuzz --output-on-failure -j $(nproc) --timeout 28800"
+      - ./tests/fuzz/calcTotalCov.sh
+
+artifacts:
+  # upload all files in the fuzz_coverage_report directory
+  files:
+    - '**/*'
```

Comment:
The broad file mask should keep this from failing, but have you tested the use of artifact uploads? (they have been flaky).

Reply:
This wildcard match looks a little bit suspicious to me. Won't this upload _everything_? I'd expect the pattern to be something more like `fuzz_coverage_report/*`.

Reply:
Tested uploading coverage files to S3 bucket and is working. I uploaded a folder `test-fuzz-coverage` to `s2n-build-artifacts` bucket and you should be able to see the report there